docs/src/userguide.md

```diff
@@ -28,22 +28,36 @@ To construct a sparse approximation to the exact posterior, we first need to sel
 M = 15  # The number of inducing points
 z = x[1:M]
 ```
-The inducing inputs `z` imply some latent function values `u = f(z)`, sometimes called pseudo-points. The stochastic variational Gaussian process (SVGP) approximation is defined by a variational distribution `q(u)` over the pseudo-points. In the case of GP regression, the optimal form for `q(u)` is a multivariate Gaussian, which is the only form of `q` currently supported by this package.
+The inducing inputs `z` imply some latent function values `u = f(z)`, sometimes called pseudo-points. The `SparseVariationalApproximation` specifies a distribution `q(u)` over the pseudo-points. In the case of GP regression, the optimal form for `q(u)` is a multivariate Gaussian, which is the only form of `q` currently supported by this package.
 ```julia
 using Distributions, LinearAlgebra
 q = MvNormal(zeros(length(z)), I)
 ```
 Finally, we pass our `q` along with the inputs `f(z)` to obtain an approximate posterior GP:
 ```julia
 fz = f(z, 1e-6)  # 'observe' the process at z with some jitter for numerical stability 
-approx = SVGP(fz, q)  # Instantiate everything needed for the svgp approximation
+approx = SparseVariationalApproximation(fz, q)  # Instantiate everything needed for the approximation
 
-svgp_posterior = posterior(approx)  # Create the approximate posterior
+sva_posterior = posterior(approx)  # Create the approximate posterior
 ```
 
 ## The Evidence Lower Bound (ELBO)
 The approximate posterior constructed above will be a very poor approximation, since `q` was simply chosen to have zero mean and covariance `I`. A measure of the quality of the approximation is given by the ELBO. Optimising this term with respect to the parameters of `q` and the inducing input locations `z` will improve the approximation.
 ```julia
-elbo(SVGP(fz, q), fx, y)
+elbo(SparseVariationalApproximation(fz, q), fx, y)
 ```
 A detailed example of how to carry out such optimisation is given in [Regression: Sparse Variational Gaussian Process for Stochastic Optimisation with Flux.jl](@ref). For an example of non-conjugate inference, see [Classification: Sparse Variational Approximation for Non-Conjugate Likelihoods with Optim's L-BFGS](@ref).
+
+## An Improved Parametrisation
```

Comment:
It's not strictly an improvement (otherwise we shouldn't bother even implementing the non-whitened version!), and I've come across some work comparing centred vs non-centred representation (e.g. http://proceedings.mlr.press/v119/gorinova20a, even if in a slightly different context) that would make me wary to say "always whiten" - even though I do think it'd be a good default!

So this is one reason why I'd prefer the two parametrisations to be the same type - then we can make it easy for people to choose either, but for the default (when you don't provide it explicitly) to be whitened.

Reply:
> It's not strictly an improvement (otherwise we shouldn't bother even implementing the non-whitened version!), and I've come across some work comparing centred vs non-centred representation (e.g. http://proceedings.mlr.press/v119/gorinova20a, even if in a slightly different context) that would make me wary to say "always whiten" - even though I do think it'd be a good default!

Excellent point -- I'll add a link to that paper in the docs and soften the language.

> So this is one reason why I'd prefer the two parametrisations to be the same type - then we can make it easy for people to choose either, but for the default (when you don't provide it explicitly) to be whitened.

I see. I think we could implement this in a couple of ways, but I agree that using a type parameter is probably the most straightforward way for now. Will refactor and see what it looks like.

Reply:
I think I've addressed this, so will resolve.